### PR TITLE
feat!(ios): Remove CAPWebView from Capacitor

### PR DESCRIPTION
**Pull request by Steven0351**
feat!(ios): Remove Portals-specific CAPWebView from Capacitor. Make various APIs public so that is possible to construct outside of Capacitor itself

### CODE REVIEW

File: ios/Capacitor/Capacitor/CAPBridgeDelegate.swift [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-internal protocol CAPBridgeDelegate: AnyObject {
+public protocol CAPBridgeDelegate: AnyObject {
```
Comment (markemer) [inline ios/Capacitor/Capacitor/CAPBridgeDelegate.swift:3]: I really want to get rid of this class, for cap 6 maybe? I don't really see the point of it. We should at least change the name, it's not a delegate.

File: ios/Capacitor/Capacitor/CAPBridgeViewController.swift [hunk 2]
```diff
@@ -92,7 +92,7 @@ import Cordova
     }
 
     open func router() -> Router {
-        return _Router()
+        return CapacitorRouter()
```
Comment (markemer) [inline ios/Capacitor/Capacitor/CAPBridgeViewController.swift:95]: 👍

**Review by markemer — APPROVED** [review]
I like it a lot - maybe want to change that protocol, but that should be a separate PR.

**Comment by markemer** [comment]
@Steven0351 You need to check in the changed project file - that's why the build is failing.

File: ios/Capacitor/Capacitor/CAPBridgeDelegate.swift (hunk 1 above)
Comment (Steven0351) [reply ios/Capacitor/Capacitor/CAPBridgeDelegate.swift:3]: Yeah, and it was also internal prior to this tbh

**Comment by Steven0351** [comment]
> @Steven0351 You need to check in the changed project file - that's why the build is failing.

I actually forgot to check-in the changes to the pbxproj

File: ios/Capacitor/Capacitor/CAPBridgeDelegate.swift (hunk 1 above)
Comment (giralte-ionic) [reply ios/Capacitor/Capacitor/CAPBridgeDelegate.swift:3]: @markemer you want to make that change in a separate PR?

**Review by giralte-ionic — APPROVED** [review]

File: ios/Capacitor/Capacitor/CAPBridgeDelegate.swift (hunk 1 above)
Comment (markemer) [reply ios/Capacitor/Capacitor/CAPBridgeDelegate.swift:3]: I think we actually might just need to rename it, I'm looking into how we use this now.

**Pull request merged**